AdminNotificationManagerPlugin.inc.php

```diff
@@ -146,19 +148,31 @@ function getTemplatePath($inCore = false) {
 	 * @return array
 	 */
 	function _getAdminList() {
-		$user = null;
+		$arrayOfContexts = $this->_getContexts();
 		$users = array();
+		$userId = array();
+		$user = null;
+		$userFactory = new PKPUserService();
 
-		$roleDAO = DAORegistry::getDAO('RoleDAO');
-		$userDAO = $roleDAO->getUsersByRoleId(ROLE_ID_SITE_ADMIN);
+		foreach($arrayOfContexts as $context) {
```

Comment:
Here there should be a space between the `foreach` and the opening parenthesis.  Also, c.f. lines 169 and 231.